neon_data_models/models/api/node_v1/__init__.py

```diff
@@ -80,8 +80,8 @@ class TextInputData(BaseModel):
 
 class NodeKlatResponse(BaseMessage):
     msg_type: Literal["klat.response"] = "klat.response"
-    data: Dict[str, KlatResponse] = Field(type=Dict[str, KlatResponse],
-                       description="dict of BCP-47 language: KlatResponse")
+    data: Dict[str, KlatResponse] = Field(
```

Comment:
Missed in #1, the old annotation was not valid and caused OpenAPI rendering exceptions